docs/install/install_everest_helm_charts.md

```diff
@@ -90,51 +104,76 @@ Here are the steps to install Percona Everest and deploy additional database nam
-            gke-everest-test-default-pool-8bbed860-65gx   Ready    <none>   3m35s   
-            v1.30.3-gke.1969001   10.204.15.199   34.175.155.135   Container- 
-            Optimized OS from Google   6.1.100+         containerd://1.7.19
-            gke-everest-test-default-pool-8bbed860-pqzb   Ready    <none>   3m35s   
-            v1.30.3-gke.1969001   10.204.15.200   34.175.120.50    Container- 
-            Optimized OS from Google   6.1.100+         containerd://1.7.19
-            gke-everest-test-default-pool-8bbed860-s0hg   Ready    <none>   3m35s   
-            v1.30.3-gke.1969001   10.204.15.201   34.175.201.246   Container- 
-            Optimized OS from Google   6.1.100+         containerd://1.7.19
-            ```
+            ??? example "Expected output"
+                ```sh
+                kubectl get nodes -o wide
+                NAME                   STATUS   ROLES    AGE   VERSION             
+                INTERNAL-IPEXTERNAL-IP  OS-IMAGE                        KERNEL-VERSION   
+                CONTAINER-RUNTIME
+                gke-everest-test-default-pool-8bbed860-65gx   Ready    <none>   3m35s   
+                v1.30.3-gke.1969001   10.204.15.199   34.175.155.135   Container- 
+                Optimized OS from Google   6.1.100+         containerd://1.7.19
+                gke-everest-test-default-pool-8bbed860-pqzb   Ready    <none>   3m35s   
+                v1.30.3-gke.1969001   10.204.15.200   34.175.120.50    Container- 
+                Optimized OS from Google   6.1.100+         containerd://1.7.19
+                gke-everest-test-default-pool-8bbed860-s0hg   Ready    <none>   3m35s   
+                v1.30.3-gke.1969001   10.204.15.201   34.175.201.246   Container- 
+                Optimized OS from Google   6.1.100+         containerd://1.7.19
+                ```
         
-        4. To launch the Percona Everest UI and create your first database cluster, go to the IP address/port found in steps 2 and 3. In this example, the external IP address used is [http://34.175.155.135:32349](http://34.175.155.135:32349). Nevertheless, you have the option to use any node IP specified in the above steps.
+        3. To launch the Percona Everest UI and create your first database cluster, go to the IP address/port found in step 1 and 3 (if TLS is enabled). In this example, the external IP address used is [http://34.175.155.135:32349](http://34.175.155.135:32349). Nevertheless, you have the option to use any node IP specified in the above steps.
 
     === "Port Forwarding"
-        Run the following command to use `Kubectl port-forwarding` for connecting to Everest without exposing the service:
+
+        Run the following command to setup a port-forward to the Percona Everest server service:
                 
-        ```sh
-        kubectl port-forward svc/everest 8080:8080 -n everest-system
-        ``` 
+        
+        `kubectl port-forward svc/everest 8080:8080 -n everest-system`
+     
```

Comment:
Shouldn't we use a code block like the other sections? (See install with CLI page)

Reply:
Fixed